apm-agent-plugins/apm-httpclient11-plugin/pom.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>apm-agent-plugins</artifactId>
+        <groupId>co.elastic.apm</groupId>
+        <version>1.17.1-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>apm-httpclient11-plugin</artifactId>
+    <name>${project.groupId}:${project.artifactId}</name>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>11</source>
+                    <target>11</target>
```

Comment:
Trying to load the instrumentations of this module would fail of the agent is attached to a Java 7 JVM, for example. But maybe that can easily be fixed by just catching the `java.lang.UnsupportedClassVersionError` in `co.elastic.apm.agent.util.DependencyInjectingServiceLoader#instantiate(java.lang.ClassLoader, java.lang.String)`. That would be quite nice.

Reply:
I added exception handling to `DependencyInjectingServiceLoader#instantiate`